RussianFilter/sections/adservers.txt

```diff
@@ -11,6 +11,7 @@
 ||kymirasite.pro^$third-party
 ||thethi.ru^$third-party
 ||krotovroman.ru^$third-party
+||tea.pornolomka.sex^$third-party
```

Comment:
Drop the $third-party part. It is a subdomain and therefore it would be first-party.